CTP/common/ext/run_sql_by_cci.sh

```diff
@@ -102,7 +102,7 @@ function run_sql {
         (cd $testResultPath/..; upload_to_dailysrv "./$testResultName" "./qa_repository/function_cci_for_sql/y`date +%Y`/m`date +%-m`/$testResultName")
 
         if [ `cat $tmplog |grep '^CORE_FILE:' | wc -l` -gt 0 ]; then
-            timestamp=`echo $testResultName|awk -F '_' '{print $(NF-2)}'`
+            timestamp=`cat $testResultPath/summary_info |grep execute|awk -F '=' '{print $(NF)}'|tr -d '[[:space:]]'|sed 's/-//g'|sed 's/\://g'`
```

Comment:
I am not sure this format is same as java's java.text.SimpleDateFormat.format, but I think it will be better to use

ss=`cat $testResultPath/summary_info |grep execute_date|awk -F '=' '{print $(NF)}'`
datestr=`date -d "$ss" +%Y%m%d%H%M%S`